flask_rebar/swagger_generation/swagger_generator.py

```diff
@@ -367,7 +367,8 @@ def generate(
             host=None,
             schemes=None,
             consumes=None,
-            produces=None
+            produces=None,
+            order_dicts=True,
```

Comment:
Nitpick, but I think this should be named `sort_keys`. This is subjective, but I feel like that's easier to understand as a user of this interface. Let me know if you disagree - my instincts are not always correct 😬 

It should also be added to the docstring.

Reply:
Done in e8dce7c. I actually had the same thought myself, but went with `order_dicts` to better match the existing `_recursively_order_dicts(...)` method. I changed that to `_recursively_sort_dicts` in e8dce7c as well. Make sense?

Reply:
Yep! that's better